Dockerfile

```diff
@@ -1,11 +1,17 @@
-
-# renovate: datasource=docker depName=ubuntu versioning=docker
 ARG UBUNTU_VERSION=bionic
+ARG BASE_IMAGE=ubuntu:${UBUNTU_VERSION}
+ARG USER_NAME=ubuntu
+ARG USER_ID=1000
+ARG APP_ROOT=/usr/src/app
 
 #--------------------------------------
 # base image
 #--------------------------------------
-FROM ubuntu:${UBUNTU_VERSION} as base
+FROM ${BASE_IMAGE} as base
+
+ARG USER_NAME
+ARG USER_ID
+ARG APP_ROOT
```

Comment:
I think the values need repeated here, otherwise they will be empty instead of reusing the above values

Reply:
I think leaving the value off works and is the recommended way